vms/proposervm/state/block_height_index.go

```diff
@@ -24,6 +24,7 @@ var (
 )
 
 type HeightIndexGetter interface {
+	GetMinimumHeight() (uint64, error)
```

Comment:
nit: please explaing what this function should do with a comment. This is not part of any other interface (like GetBlockIDAtHeight)